src/kw_env.sh

```diff
@@ -72,6 +77,25 @@ function use_target_env()
   printf '%s\n' "$target_env" > "${local_kw_configs}/${ENV_CURRENT_FILE}"
 }
 
+function use_no_env()
+{
+  local current_env
+  local local_kw_configs="${PWD}/.kw"
+
+  current_env=$(< "${local_kw_configs}/${ENV_CURRENT_FILE}")
+
+  if [[ $(ask_yN 'Do you want to make this env the default?') =~ '1' ]]; then
+    for config in "${config_file_list[@]}"; do
+      if [[ -L "${local_kw_configs}/${config}.config" ]]; then
```

Comment:
Add a comment before this `if` that explains that you are removing the link but not the original file.